cypress/integration/backend-admin-ui/test_2_dataset.js

```diff
@@ -61,20 +61,24 @@ context("Admin UI Single Dataset", () => {
         });
     });
 
-    describe("Can delete test table and concept", () => {
+    describe("Can delete test concept", () => {
+        before(() => { visitAdminUI(`datasets/${testDSID}`); });
 
         it("Can delete test concept", () => {
             cy.get('[data-test-id="accordion-Concepts"]').click();
             cy.get(`[data-test-id="delete-btn-concept-${testDSID}.concept1"]`).click({force: true});
             cy.get(`[data-test-id="delete-btn-${testDSID}.concept1"]`).should('not.exist');
         });
+    });
+
+    describe("Can delete test table", () => {
```

Comment:
If you know put both `it` back in one `describe` it should have the same effect, doesn't it?

Reply:
Yes, but only if `visitAdminUI` is called after the test concept got deleted. Therefore i changed `before` to 'beforeEach` to make sure the page does reload before the deletion of the test table and combined both tests under one `describe` again.
If you got nothing more to add i would restart the e2e job multiple times again to make sure it works and merge the pr afterwards.